src/Blt/Plugin/Commands/CohesionCommands.php

```diff
@@ -53,8 +53,14 @@ public function postCommand($result, CommandData $commandData)
   {
     $rebuild = $this->getConfigValue('site-studio.rebuild');
     $sync_import = $this->getConfigValue('site-studio.sync-import');
+    $package_import = $this->getConfigValue('site-studio.package-import');
     $cohesion_import = $this->getConfigValue('site-studio.cohesion-import');
 
+    // Check if both package_import and sync_import are enabled.
+    if (isset($sync_import) && $sync_import && isset($package_import) && $package_import) {
+      throw new BltException("Cannot use both sync import and package import simultaneously. Use one or the other.");
```

Comment:
This exception is only thrown if $sync_import is defined and TRUE, but here https://github.com/davidtrainer/blt-site-studio/pull/25/files#diff-58e6b6e439a8f488c64e1e001d23b780b1b6a281f8f21f13033be77c438c90c2R100 we go ahead and run sync:import if $sync_import is undefined.

What should be the default behavior if someone requires the plugin but doesn't configure anything? I think we should default to using package:import, and not sync:import, at least on the 2.x branch.

Reply:
Yes, if we are creating a new branch to support 6.8 I agree. Just made it this way so it didn't break existing assumptions/functionality.